vendor/bosch/tps110-0233.yaml

```diff
@@ -1,5 +1,5 @@
 id: tps110-0223
-vendorProfileID: 1
+vendorProfileID: 69
```

Comment:
Did they provide this?

Reply:
Took from the list you shared: https://github.com/TheThingsNetwork/lorawan-devices/issues/5#issuecomment-648782960